services/ircd/inspircd/conf.nix

```diff
@@ -333,7 +328,6 @@ in {
     maxlines="20";
     notice="yes";
   }];
-  autojoin = [{ channel="#lobby,#helpdesk"; }];
```

Comment:
Is this handled by something else? Because I would say we should still autojoin lobby.

Reply:
no but given oldsoc I don't think we actually want it